src/SFA.DAS.ApprenticeFeedback.Database/StoredProcedures/GenerateFeedbackTransactions.sql

```diff
@@ -0,0 +1,21 @@
+﻿CREATE PROCEDURE [dbo].[GenerateFeedbackTransactions]
+
+AS
+
+DECLARE @CreatedOn datetime
+SELECT @CreatedOn = GETUTCDATE()
+
+INSERT INTO dbo.FeedbackTransaction (ApprenticeFeedbackTargetId, CreatedOn)
+SELECT aft.Id, @CreatedOn
+FROM [dbo].[ApprenticeFeedbackTarget] aft
+LEFT JOIN (SELECT ApprenticeFeedbackTargetId
+           FROM [dbo].[FeedbackTransaction] 
+           WHERE SentDate IS NULL OR (SentDate IS NOT NULL AND SentDate >= DATEADD(day,-90,GETDATE())) ) ft1
```

Comment:
The 90 days passed in for the sent date needs to be a configurable parameter passed in from the application settings

Reply:
Changed to use app setting passed into stored proc